TTITransition/ViewControllers/NewsTableViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  NewsTableViewController.swift
+//  TTITransition
+//
+//  Created by 1 on 05.04.2018.
+//  Copyright © 2018 ANDRE.CORP. All rights reserved.
+//
+
+import UIKit
+
+class NewsTableViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+    
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+
+    // MARK: - Table view data source
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! NewsTableViewCell
```

Comment:
force unwrapp!! it will be crash if it cant cast to this class. Use "guard" instead of force unwrapping. if it is nil return UITableViewCell - empty cell or use assertFailure() - learn it too